packages/studio/src/store/useStudioStore.ts

```diff
@@ -56,6 +56,10 @@ const useStudioStore = create<StudioStore>()(
 );
 
 void useStudioStore.getState().actions.refreshActivePageEntities();
+void (
+  useStudioStore.getState().studioConfig.isPagesJSRepo &&
+  useStudioStore.getState().accountContent.refreshBaseAccountContent()
+);
```

Comment:
Until we add the logic for sourcing the user's API key in `studio-plugin`, we will see the following error in the browser console. If we'd prefer not to have the error appear in the interim, I can remove this call to `refreshBaseAccountContent`

<img width="787" alt="Screenshot 2023-09-06 at 3 38 43 PM" src="https://github.com/yext/studio/assets/88398086/acfda407-feb3-4f41-bfa4-7454d8d00061">

Reply:
is there a way we could catch the error?

Reply:
Updated to catch the error, so now the console has these logs instead:
<img width="766" alt="Screenshot 2023-09-07 at 9 10 03 AM" src="https://github.com/yext/studio/assets/88398086/b1743a4a-f2f5-47b0-a221-8f816bdbcaa9">